guide/case-studies/payjoin.md

```diff
@@ -0,0 +1,333 @@
+---
+layout: guide
+title: The Payjoin experience
+description: This case study is an analysis and exploration of payjoin's audience, implementations, features and proposes specific user flows.
+nav_order: 6
+parent: Case studies
+permalink: /guide/case-studies/payjoin/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-preview.jpg
```

Comment:
```suggestion
image: "assets/images/guide/case-studies/payjoin-case-study/header-graphic-metaphor-preview-image.png"
```

Reply:
You probably don't want the `"` at the start and end of the file name